### PR TITLE
Fix classes generation with YAML or XML metadata.

### PR DESCRIPTION
**Pull request by baiolo**
The documentation misses the change of one configuration file if the generated metadata is in YAML or XML format. This PR documents how to change the configuration file.

### CODE REVIEW

File: doctrine/reverse_engineering.rst [hunk 1]
```diff
@@ -85,6 +85,24 @@ run:
     // generates getter/setter methods
     $ php bin/console make:entity --regenerate App
 
+If you generated XML or YAML metadata you have also to change the ``config/packages/doctrine.yaml``
+file like this (the example is for YAML):
+
+.. code-block:: yaml
+
+
```
Comment (OskarStark) [inline doctrine/reverse_engineering.rst]: please remove this blank line

File: doctrine/reverse_engineering.rst [hunk 2]
```diff
@@ -85,6 +85,24 @@ run:
...
+    doctrine:
+        ...
```
Comment (OskarStark) [inline doctrine/reverse_engineering.rst]: this is invalid yaml, please use `# ...`

Comment (baiolo) [reply doctrine/reverse_engineering.rst]: Hi @OskarStark, do you mean something like this ?
```
    doctrine:
        ###
        orm:
            ###
            mappings:
                App:
                    is_bundle: false
                    type: yml # Previously was 'annotation'
                    dir: '%kernel.project_dir%/config/doctrine' # Previously was '%kernel.project_dir%/src/Entity'
                    prefix: 'App\Entity'
                    alias: App
```
If not how i can shorten the above example ?

Comment (baiolo) [reply doctrine/reverse_engineering.rst]: Forget my last comment, it should be:
```
    doctrine:
        # ...
        orm:
            # ...
            mappings:
                App:
                    is_bundle: false
                    type: yml # Previously was 'annotation'
                    dir: '%kernel.project_dir%/config/doctrine' # Previously was '%kernel.project_dir%/src/Entity'
                    prefix: 'App\Entity'
                    alias: App
```

File: doctrine/reverse_engineering.rst [hunk 3]
```diff
@@ -85,6 +85,23 @@ run:
     // generates getter/setter methods
     $ php bin/console make:entity --regenerate App
 
+If you generated XML or YAML metadata you have also to change the ``config/packages/doctrine.yaml``
```
Comment (xabbuh) [inline doctrine/reverse_engineering.rst]: Wouldn't this better fit in the `tip` block above where we show that you can run `doctrine:mapping:import` with xml or yml?

Comment (xabbuh) [reply doctrine/reverse_engineering.rst]: I mean we can update it to look like this:

```rst
.. tip::

    It's also possible to generate the metadata files into XML or YAML:

    .. code-block:: terminal

        $ php bin/console doctrine:mapping:import 'App\Entity' xml --path=config/doctrine

    In this case, make sure to adapt your mapping configuration accordingly:

    .. code-block:: yaml

        # config/packages/doctrine.yaml
        # ...
```

File: doctrine/reverse_engineering.rst [hunk 4]
```diff
@@ -70,6 +70,22 @@ files: ``BlogPost.php`` and ``BlogComment.php``.
 
         $ php bin/console doctrine:mapping:import 'App\Entity' xml --path=config/doctrine
 
+    In this case, make sure to adapt your mapping configuration accordingly:
+
+    .. code-block:: yaml
+
+        doctrine:
```
Comment (xabbuh) [inline doctrine/reverse_engineering.rst]: Can you add the `# config/packages/doctrine.yaml` filename comment above this line?

**Review by xabbuh — APPROVED** [review]

**Pull request closed without merging**